charge_box.classes/LogFileSimulator.cs

```diff
@@ -0,0 +1,46 @@
+﻿namespace charge_box.classes
+{
+    public class LogFileSimulator : ILogFile
+    {
+        public string FilePath_ = Environment.CurrentDirectory;
+        public void LogDoorLocked(int id, DateTime TimeOfEvent)
+        {
+            string message = "Door Has been Locked: " + TimeOfEvent;
+            TimeOfEvent = GetCurrentTime();
```

Comment:
Metoden for sin TimeOfEvent med som parameter. 
Det gør den fordi det er tiden når event sker som er interessant. 
Ved at overskriver med en ny getCurrentTime(), bliver time of event i realitten til timeOfLogging. 

Jeg tro simplenthend bare den skal gennems direkte

Reply:
Det var også en linje der skulle være slettet :) Tak for skarpe øjne